APIManager.js

```diff
@@ -0,0 +1,67 @@
+class APIManager {
+    constructor(renderer) {
+        this.renderer = renderer
```

Comment:
instead of storing the renderer here, you should store `this.data` and when you create the renderer instance in the main.js you should invoke it with `api.data`

this is because your logic should not 'talk' to your renderer. The controller should invoke the APIManager and send the data gathered from the API to the renderer.